README.md

```diff
@@ -26,7 +26,30 @@ For development purposes it is possible to deploy the function to GCP from a loc
 
 If this is the first time deploying a Cloud Function to a project, the Cloud Build and Cloud Functions APIs may need to be enabled - navigate to `https://console.developers.google.com/apis/library/cloudbuild.googleapis.com?project=your-project-name` and `https://console.developers.google.com/apis/library/cloudfunctions.googleapis.com?project=your-project-name` to enable them.
 
-Once authenticated, run `NOTIFY_API_KEY=<your-api-key> make deploy_function`.
+For the cloud function to work it needs a valid notify_api_key set, this can be done in 2 ways
+
+**Secret Manager in GCP**
+
+Firstly check that Secret Manager in GCP is enabled, if not enable it. You can add the notify_api_key manually in the UI and update `App Engine default service account` to have `Secret Manager Secret Accessor` to the new notify_api_key.
+
+or you can run the following gcloud commands
+
+```
+gcloud secrets create notify_api_key --data-file=<data_file> --project=<project_id> --replication-policy=<replication-policy> --locations=<locations>
+
+gcloud secrets add-iam-policy-binding notify_api_key --role roles/secretmanager.secretAccessor --member serviceAccount:<project_id>@appspot.gserviceaccount.com
+```
+
+N.B. replication-policy can be `automatic` or `user-managed`. If automatic neither `replication-policy` or `location` needs to be included in the command, if user-managed `replication-policy` and `location` must be provided
```

Comment:
Although fine, prob only want to document what we will actually use, which I believe is always London region. So this isn't needed?

Reply:
We/someone else might in the future so I dont think it hurts to leave as is